.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "arrowParens": "avoid",
+  "bracketSpacing": true,
+  "endOfLine": "auto",
+  "printWidth": 120,
+  "singleQuote": false,
+  "tabWidth": 2,
+  "trailingComma": "all"
+}
```

Comment:
lets change this to the config in the previous yaml file to not have a bunch of style changes in this PR

Reply:
I think I got this from mento-core-ts and thought that I'd make the style consistent in all repos but I'll revert it then